test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/BindingSourceModelBinderTest.cs

```diff
@@ -90,7 +90,7 @@ public void BindingSourceModelBinder_ThrowsOnNonGreedySource()
             var result = await binder.BindModelAsync(context);
 
             // Assert
-            Assert.NotNull(result);
+            Assert.NotEqual(ModelBindingResult.NoResult, result);
```

Comment:
This test is failing in the CI builds. Probably now two separate tests: In one case the system returns `NoResult` and in the other, the model is set.